### ISSUE TITLE
Issues with interpolation on a Sentinel Tie Point Grid. 

### ISSUE THREAD

**Issue by BenConstable9** [pytroll/python-geotiepoints#37]
#### Code Sample, a minimal, complete, and verifiable piece of code

```python
tie_cols = np.arange(
    1, image_data.shape[0], math.ceil(image_data.shape[0] / longitude.shape[1])
)
print(tie_cols.shape)
fine_cols = np.arange(image_data.shape[1])

tie_rows = np.arange(
    1, image_data.shape[1], math.ceil(image_data.shape[1] / latitude.shape[0])
)
print(tie_rows.shape)
fine_rows = np.arange(image_data.shape[0])

interpolator = GeoInterpolator(
    (longitude, latitude), (tie_rows, tie_cols), (fine_rows, fine_cols)
)

return interpolator.interpolate()

```
#### Problem description

I'm trying to interpolate a Tie Point Grid from a Sentinel SAR1A image but having issues with the results. The original tie grid is 10 x 21 and I'm trying to increase it to the same number of pixels as the image. 

The code runs fine but visually checking the image location against the given pixel grid point gives the wrong coordinates. Is something I've doing obviously wrong?

I'd appreciate any guidance and thanks for spending time on the library.

#### Versions of Python, package at hand and relevant dependencies

Thank you for reporting an issue !

**Comment by mraspaud** [comment]
Thanks a lot for reaching to us about this issue!
I am a little familiar with S1A data, and I am curious as to whether the longitudes and latitudes you have are really on a grid? I've notice that the tiepoints in the GRD data at least are sometimes shifted after a couple of rows...

**Comment by mraspaud** [comment]
No, this should be fine. What I mean is that every tie point has an associated line and column number assigned to it. The way you build the grid, you assume that all the coordinates of the first column of your array are on the same column on the image, which in my experience is most of the times not the case for that type of data.

For example, I have seen case like tuples of (line, col, lon, lat) would be like
(0, 0, ..., ...)
(10, 0, ..., ...)
(20, 1, ..., ...)
(30, 1, ..., ...)
(0, 10, ..., ...)
(10, 10, ..., ...)
(20, 10, ..., ...)
(30, 11, ..., ...)
etc

I'm not saying this is the case for you data, but it happened to me a lot with Sentinel 1 data.

**Comment by mraspaud** [comment]
If you are using H5 data then it's probably not the same than I am using. Could you produce the line and column numbers for each lon lat points so we would have a full dataset to test?

**Comment by djhoese** [comment]
It looks like interest in this problem died off. I'm going to close this unless this is still a problem and we have more info to start figuring out what is going on. Anyone feel free to comment and we can reopen this.
